docs/en/reference/sql/ddl/CREATE_TABLE_STATEMENT.md

```diff
@@ -1,40 +1,35 @@
 # CREATE TABLE
+ The `CREATE TABLE` statement is used to create a table. The table name must be unique in one database. If the table with the same name in a database is created repeatedly, an error will occur.
 
 ## Syntax
 
 ```sql
 CreateTableStmt ::=
     'CREATE' 'TABLE' IfNotExists TableName ( 
       TableElementList CreateTableSelectOpt | LikeTableWithOrWithoutParen ) OnCommitOpt
-
 IfNotExists ::=
     ('IF' 'NOT' 'EXISTS')?
-
 TableName ::=
     Identifier ('.' Identifier)?
     
 TableElementList ::=
     TableElement ( ',' TableElement )*
-
 TableElement ::=
-    ColumnDef
-|   ColumnIndex
+    ColumnDef | ColumnIndex
 ```
 
- The `CREATE TABLE` statement is used to create a table. The table name must be unique if it's in the same database. If the table with the same name is created repeatedly, an error will occur.
 
-The `table_element` list needs to be defined in the table creation statement. `table_element` is divided into column description `ColumnDef` and `Constraint`. OpenMLDB requires at least one ColumnDef in the `table_element` list.
+The `table_element` list needs to be defined in the `CREATE TABLE` statement. `table_element` is divided into column description `ColumnDef` and `Constraint`. OpenMLDB requires at least one `ColumnDef` in the `table_element` list.
```

Comment:
what is table_element?
is it that TableElementList?

Reply:
`table_element` list is equal TableElementList

Reply:
there is no context about table_element, should change to TableElementList

Reply:
ok
I will change it